### DISCUSSION TITLE
New Activity Bar Top Position Theming Information?

### DISCUSSION THREAD

**Discussion by AndreasBackx** [microsoft/vscode-discussions#912] [Extension Development QnA] [answered]
I am trying to update my [Rubecula](https://marketplace.visualstudio.com/items?itemName=andreasbackx.rubecula) theme to support the new activity bar top placement. I'm having a hard time finding what the theme keys are according to the [Theme Color](https://code.visualstudio.com/api/references/theme-color) page. Does anyone know?

Additionally, is there any functionality (or an issue requesting it) to inspect these theme keys like we have for other editor functionality? (Context keys, edit tokens & scopes...)

![image](https://github.com/microsoft/vscode-discussions/assets/1593486/ed8dd63e-fea6-4e1b-a338-c02c375681ac)
![image](https://github.com/microsoft/vscode-discussions/assets/1593486/1a3c9a3d-fca9-4073-a46f-06605df31e60)

**Comment by isidorn** [comment]
@sandy081 should be able to help here

**Reply by sandy081** [reply, marked as answer]
Please follow up on this issue - https://github.com/microsoft/vscode/issues/197345

**Reply by AndreasBackx** [reply]
@sandy081 thanks! Then I guess it means this is not supported (yet).
